### PR TITLE
Edit reminder

### LINKED ISSUE #6: Display a message if there are no reminders
When viewing an existing reminder, the user can click on an “Edit” button where they can modify the title, notes, or date of the reminder, with a “Save” button.

In a previous issue you created a form to add a new reminder. This issue will require duplicating a lot of that code. Instead, refactor how your app is structured to use a `reminder-form` component to both add and edit a reminder.

You will need to move some logic around to handle where your actions are fired, and how the form gets information if the fields need to be pre-populated.

**Hints**:

* Ember logic bubbles from the controller to the route (the direct order of communication goes template --> controller --> component --> route ).
* Use `ember g component reminder-form` to generate your component.
* You might need to adjust your file structure.
* In terms of UX, when the user hits the save button, they should be returned to view the individual reminder they just edited.

**Hot Tip**: To edit a reminder you need to know which reminder you are talking about. Think about this when you are structuring your `router.js` file.  

Additional Helpful Resources:
[Ember Best Practice: Stop Bubbling and Use Closure Actions](https://dockyard.com/blog/2015/10/29/ember-best-practice-stop-bubbling-and-use-closure-actions)
[Closure Actions in Ember](https://emberway.io/route-closure-actions-in-ember-js-d0a7a37a5d1b#.ko5ueszgy)

### PR DESCRIPTION
**Pull request by Obleo33**
## Purpose

Add edit reminder functionality 

closes #6 

## Approach

Add edit button to individual reminder view
When edit is clicked the values are put into a form that you are able to edit

### Learning

Taylor walkthrough

### Test coverage 

Test click on the first reminder => click on the edit button => change the title and body => click submit => confirm that the title and body have been updated. 

### Follow-up tasks

#7

### CODE REVIEW

**Comment by Obleo33** [comment]
@Tman22 @martensonbj PR up for edit reminder

File: app/controllers/reminders/new.js [hunk 1]
```diff
@@ -7,6 +7,6 @@ export default Ember.Controller.extend({
       model.save().then(()=> {
         this.transitionToRoute('reminders.reminder', model)
       })
-    }
+    },
```
Comment (Tman22) [inline app/controllers/reminders/new.js]: ?

File: app/routes/reminders/reminder/edit.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  actions:  {
```
Comment (Tman22) [inline app/routes/reminders/reminder/edit.js]: Take this out if you're not using it.

File: app/components/reminder-form.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+
```
Comment (Tman22) [inline app/components/reminder-form.js]: What's this file doing?

File: app/controllers/reminders/new.js (hunk 1 above)
Comment (Obleo33) [reply app/controllers/reminders/new.js]: we had more than one action saved on new.js but the other was removed. I missed the comma

File: app/routes/reminders/reminder/edit.js (hunk 2 above)
Comment (Obleo33) [reply app/routes/reminders/reminder/edit.js]: removed

File: app/components/reminder-form.js (hunk 3 above)
Comment (Obleo33) [reply app/components/reminder-form.js]: removed

**Comment by Obleo33** [comment]
@Tman22 updated

**Pull request merged**